packages/fluentui/docs/src/examples/components/Datepicker/Types/DatepickerExampleWeek.shorthand.steps.ts

```diff
@@ -0,0 +1,19 @@
+import { ScreenerTestsConfig } from '@fluentui/scripts/screener';
+import { buttonClassName, datepickerCalendarCellClassName } from '@fluentui/react-northstar';
+
+const config: ScreenerTestsConfig = {
+  themes: ['teams', 'teamsDark', 'teamsHighContrast'],
+  steps: [
+    builder =>
+      builder
+        .click(`.${buttonClassName}`)
+        .snapshot('Shows calendar.')
+        .hover(`.${datepickerCalendarCellClassName}:nth-child(1)`)
+        .snapshot('Calendar is opened with the entire week highlighted on hover.')
+        .click(`.${datepickerCalendarCellClassName}:nth-child(1)`)
+        .click(`.${buttonClassName}`)
+        .snapshot('Calendar is opened with selected week highlighted.'),
```

Comment:
This screenshot seems redundant

![image](https://user-images.githubusercontent.com/20744592/107568301-b2b43b80-6bde-11eb-8625-cff9651e26a3.png)


```suggestion
        .click(`.${buttonClassName}`)
        .hover(`.${datepickerCalendarCellClassName}:nth-child(1)`)
        .snapshot('Calendar is opened with the entire week highlighted on hover.')
        .click(`.${datepickerCalendarCellClassName}:nth-child(1)`)
        .click(`.${buttonClassName}`)
        .snapshot('Calendar is opened with selected week highlighted.'),
```